### ISSUE TITLE
get-pkg is broken

### ISSUE THREAD

**Issue by stefanwalther** [jonschlinkert/get-pkg#4]
It seems, that something has changed on npm, basically the functionality and error handling in this repo is broken, which also causes verb-generate-readme to discontinue to work.

Running the tests of this repo (as of today) returns the following:

```
getPackages
    1) should get a package.json for the given project
    2) should handle errors
    3) should handle empty string
    4) should handle scoped packages


  0 passing (174ms)
  4 failing

  1) getPackages should get a package.json for the given project:
     Uncaught SyntaxError: Unexpected end of JSON input
      at JSON.parse (<anonymous>)
      at index.js:37:20
      at IncomingMessage.<anonymous> (node_modules/min-request/request.js:68:4)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)

  2) getPackages should handle errors:
     Uncaught SyntaxError: Unexpected end of JSON input
      at JSON.parse (<anonymous>)
      at index.js:37:20
      at IncomingMessage.<anonymous> (node_modules/min-request/request.js:68:4)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)

  3) getPackages should handle empty string:
     Uncaught SyntaxError: Unexpected end of JSON input
      at JSON.parse (<anonymous>)
      at index.js:37:20
      at IncomingMessage.<anonymous> (node_modules/min-request/request.js:68:4)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)

  4) getPackages should handle scoped packages:
     Uncaught SyntaxError: Unexpected end of JSON input
      at JSON.parse (<anonymous>)
      at index.js:37:20
      at IncomingMessage.<anonymous> (node_modules/min-request/request.js:68:4)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)


```

**Comment by stefanwalther** [comment]
The problem is that npm returns a "301 - Moved permanently" which is not handled properly by `min-request`:

![image](https://user-images.githubusercontent.com/669728/40579939-8560735e-6134-11e8-97af-a986ae3976e3.png)

**Comment by stefanwalther** [comment]
I have submitted a PR, see #5

**Comment by stefanwalther** [comment]
@jonschlinkert & @doowb Could you pls verify ...
I have tried to get this repo working again, but it's really hard to update all the dependencies to get verb-generate-readme up and working again, also I don't know if the tests in this repo really cover all of the requirements when using `get-pkg` in `generate-verb-readme` and other `verbose` related repositories.

**Comment by jonschlinkert** [comment]
closed by 1.0. Thanks for the pr @stefanwalther!
